_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -14,6 +15,14 @@
 
 
 // ----------------------------
+// #PATHS
+// ----------------------------
+$static-path: '..' !default;
+$font-path: '#{$static-path}/fonts' !default;
+$image-path: '#{$static-path}/images' !default;
+
```

Comment:
I'm 51/49 on changing it. I say that because, since this is our Pattern Library, I'd kind of like to keep it streamlined. If another person or organization needed to change the structure, I think we can safely assume they could easily create another variable as needed. Though, it is nice seeing the usage example on lines 21 and 22.